### PR TITLE
STORM-1888: add description for shell command

### PR DESCRIPTION
**Pull request by vesense**
This can be cherry-picked to 1.x-branch.

### CODE REVIEW

File: bin/storm.py [hunk 1]
```diff
@@ -485,6 +485,11 @@ def kill_workers(*args):
         extrajars=[USER_CONF_DIR, os.path.join(STORM_DIR, "bin")])
 
 def shell(resourcesdir, command, *args):
+    """Syntax: [storm shell resourcesdir command args]
+
+    Makes constructing jar and uploading to nimbus for using non JVM languages
```
Comment (HeartSaVioR) [inline bin/storm.py]: I feel this line misses behavior "executing command".

I'd like to have this line changed like 
`Archives resources to jar and uploads jar to Nimbus, and executes following arguments on "local". Useful for non JVM languages.`

Comment (vesense) [inline bin/storm.py]: Thanks @HeartSaVioR This is even better. Updated.

**Comment by vesense** [comment]
@HeartSaVioR Updated and rebased.

**Comment by HeartSaVioR** [comment]
+1 Thanks @vesense for following up.

**Pull request merged**